scripts/run-telepresence.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script executes telepresence against an operator deployment.
+#
+#   KUBECONFIG=... TP_DEPLOYMENT_YAML=... TP_CMD_PATH=... run-telepresence.sh
+#
+# The script is parameterized by env vars prefixed with 'TP_'.
+#
+# Dependencies:
+#
+# - oc
+# - yq (pip install yq)
+# - jq (fedora: dnf install jq; macos: brew install jq)
+# - telepresence
+#    - fedora:
+#      - curl -s https://packagecloud.io/install/repositories/datawireio/telepresence/script.rpm.sh | sudo bash
+#      - sudo dnf install telepresence
+#    - macos:
+#      - brew cask install osxfuse # reboot required
+#      - brew install datawire/blackbird/telepresence
+#    - Current release (0.104) not compatible with ocp.
+#      - Workaround: Install dependencies via package but telepresence from source
+#        - git clone https://github.com/telepresence/telepresence
```

Comment:
You mean this one `https://github.com/telepresenceio/telepresence`?

Reply:
Good catch, done.